src/pages/home/_Features.tsx

```diff
@@ -1,220 +1,11 @@
-                    >
-                        <small className="text-uppercase font-weight-bold d-block mb-2">{feature.productFeature}</small>
-                        <h2 className={classNames('font-weight-bold', { 'max-w-350': index !== 0 })}>
-                            {feature.title}
-                        </h2>
-                        <p>{feature.description}</p>
-                        <ul>
-                            {feature.details.map(detail => (
-                                <li key={detail}>{detail}</li>
-                            ))}
-                        </ul>
-                        {feature.ctaLink.includes('http') ? (
-                            <a
-                                href={feature.ctaLink}
-                                target="_blank"
-                                rel="noreferrer"
-                                className="btn btn-outline-primary mt-2"
-                                data-button-style={buttonStyle.outline}
-                                data-button-location={buttonLocation.body}
-                            >
-                                Learn more about {startCase(feature.productFeature)}
-                            </a>
-                        ) : (
-                            <Link href={feature.ctaLink} passHref={true}>
-                                <a
-                                    className="btn btn-outline-primary mt-2"
-                                    href="#none"
-                                    data-button-style={buttonStyle.outline}
-                                    data-button-location={buttonLocation.body}
-                                >
-                                    Learn more about {startCase(feature.productFeature)}
-                                </a>
-                            </Link>
-                        )}
-                    </div>
-
-                    <div className="col-lg-6 mb-6 mb-lg-0 py-0 px-5">
-                        <video
-                            className={`shadow w-100 max-w-550 d-block mx-auto rounded video-${index}`}
-                            autoPlay={false}
-                            muted={true}
-                            loop={true}
-                            playsInline={true}
-                            controls={false}
-                            data-cookieconsent="ignore"
-                        >
-                            <source type="video/webm" src={feature.video.webm} data-cookieconsent="ignore" />
-                            <source type="video/mp4" src={feature.video.mp4} data-cookieconsent="ignore" />
-                        </video>
-                    </div>
-                </div>
-            ))}
-        </ContentSection>
-    )
-}
+const FeatureSection: FunctionComponent = () => (
+    <ContentSection color="white" className="py-8 pb-7">
+        <FeatureWalkthrough />
+    </ContentSection>
+)
 
 export default FeatureSection
```

Comment:
I would remove this file since it's so small and just add the content section to the homepage index since you can import `FeatureWalkthrough` now. It's probably a bit overkill to have a separate component just for this.

```suggestion
```